### ISSUE TITLE
Error with push command since last realease

### ISSUE THREAD

**Issue by xNok** [nulldriver/cf-cli-resource#39]
Since the latest relase I am unable to use the command push properly

```
   - put: pcf
      params:
        name: simulation
        manifest: simulation/ci/manifest-dev.yml
        path: artifactory-rc/simulation-*.jar
```
I get the following error:
```
Incorrect Usage: The specified path 'artifactory-rc/simulation-*.jar' does not exist.
```
While the file does exit, also and I have ben using the same pipeline for a couple day without amy trouble

**Comment by patrickcrocker** [comment]
Oops... let me take a look at this.

**Comment by patrickcrocker** [comment]
Code fixed, just waiting for the [test job](https://ci.nulldriver.com/teams/resources/pipelines/cf-cli-resource/jobs/test/builds/20) to pass so I can release.  Just a few more minutes...

**Comment by patrickcrocker** [comment]
Fixed in [v2.8.2](https://github.com/nulldriver/cf-cli-resource/releases/tag/v2.8.2).  I've pulled the 2.8.1 release from Docker Hub.
